.github/workflows/ghc.yml

```diff
@@ -47,6 +69,20 @@ jobs:
         with:
           hlint-bin: ${{ steps.get-hlint-bin.outputs.hlint-bin }}
 
+      - uses: actions/checkout@v4
```

Comment:
@RoukayaZaki, don't use `checkout` here. There's already a step before.

Also, you need to add the second `copyrights-action`. It should check `Abs.hs` and `Print.hs` against the [LICENSE](https://github.com/objectionary/eo-phi-normalizer/blob/c53ed6a49b5632c4eed7cc347d1fb3bfc1f7361e/eo-phi-normalizer/LICENSE) file in the `eo-phi-normalizer` directory.